internal/consumer/cron.go

```diff
@@ -0,0 +1,33 @@
+package consumer
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/emmaLP/gs-software-onboarding/internal/model"
+
+	"github.com/robfig/cron/v3"
+	"go.uber.org/zap"
+)
+
+func ConfigureCron(ctx context.Context, logger *zap.Logger, config *model.Configuration) error {
+	c := cron.New()
+
+	var err error
+	storyProcessing := func() {
+		service, _ := NewService(logger, &config.Consumer, nil)
+		err = service.processStories(ctx)
+	}
```

Comment:
Dont think this needed to be a func adds complexity imo.

```
service, _ := NewService(logger, &config.Consumer, nil)

if err := service.processStories(ctx); err != nil {
	return fmt.Errorf("Error occurred processing stories: %w", err)
}
```

Reply:
The cron.AddFunc doesn't support passing a function that has a return value that's why I added the anonymous function. 

I do agree that the NewService call can be pulled out of the anonymous function